package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "6.4.2",
+  "version": "6.4.3",
```

Comment:
This is a significant cleanup/update - go for 6.5.0 !

Reply:
On it! 🚀

Reply:
Fixed, thanks.